CMakeLists.txt

```diff
@@ -400,6 +406,8 @@ if(ENABLE_MASON)
     set(ENV{CCACHE_CPP2} "true")
   endif()
 
+  include_directories(SYSTEM ${CMAKE_CURRENT_SOURCE_DIR}/third_party/libosmium/include)
```

Comment:
Done in https://github.com/Project-OSRM/osrm-backend/commit/4751b8af3c8680f3bb35e4c0d09d4d5170edd397